packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -383,15 +387,13 @@ class HeroParty {
     _currentPerformance = null;
   }
 
-  Iterable<Widget> getWidgets(BuildContext context, PerformanceView performance) sync* {
+  List<_HeroQuestState> setPerformance(PerformanceView performance) {
```

Comment:
not sure what's going on with this function, it has a return type but that looks like it was a mistake?